pandasai/helpers/df_config.py

```diff
@@ -13,6 +14,7 @@ class Config(BaseModel):
     save_logs: bool = True
     verbose: bool = False
     enforce_privacy: bool = False
+    sample_head: pd.DataFrame = None
```

Comment:
I would rather keep it as a dedicate param. Configs are most of the times settings that can be applied both to the datalake and dataframe.